Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs

```diff
@@ -22,16 +41,21 @@ public static string GetCropUrls(this IPublishedContent image, string cropAlias,
                 imageUrl = UpdateCropSettings(imageUrl, width, height, enableWebP, quality);
             }
 
-            if (!includeRetina)
+            return imageUrl;
+        }
+
+        public static string GetCropUrlRetina(this IPublishedContent image, string cropAlias = null, int? width = null, int? height = null, bool enableWebP = false, int? quality = null)
```

Comment:
IMO we should move/split those parameters to new line, they are to long in single line.